sympy/integrals/laplace.py

```diff
@@ -1450,11 +1450,16 @@ def _frac(f, s):
         except PolynomialError:
             return f
 
+    def _is_integer(x):
+        return x.is_integer
```

Comment:
What is this function for?

Reply:
There was a table entry that used it during development, but that entry is no longer there, so I just remove the function.